design/apidsl/action.go

```diff
@@ -126,6 +129,7 @@ func Routing(routes ...*design.RouteDefinition) {
 	}
 }
 
+// GET used in: Routing
```

Comment:
Maybe we should say that `Routing` does not take a DSL, `GET` (and the other HTTP method functions) is just an argument to `Routing`.